src/Function.cpp

```diff
@@ -119,6 +119,25 @@ MultiValuedFunctionAttribute::OutputAttributes(std::ostream &out)
 		OutputAttribute(out, "(\"" + attribute_values[rnd_upto(attribute_values.size())] + "\")");
 }
 
+AlignedFunctionAttribute::AlignedFunctionAttribute(string name, int prob)
+	: FunctionAttribute(name, prob)
+{
+}
+
+void
+AlignedFunctionAttribute::OutputAttributes(std::ostream &out)
+{
+	if(rnd_flipcoin(attribute_probability)){
+	int power = rnd_upto(16);
+		if(!attr_emitted){
+			out << " __attribute__((" << "aligned(" << (1 << power) << ")";
```

Comment:
This should be handled only at one place, which is probably `FunctionAttribute::OutputAttribute`.